source/_drafts/Video.js-Flash-Removed-From-Core-Player.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Video.js removes flash from core player
+tags:
+  - Video.js 6.0
+  - Flash
+  - html5
+author:
+  name: Michael Roca
+  github: mrocajr
+date: 2017-01-25 14:50:06
+---
+
+In August of 2016, we [announced] our intention of removing flash as a part of the core video.js project.
+As html5 video becomes the standard playback tech and flash fades into obsolescence, it is time 
+to remove flash from the core player and move it to a separate code base.  This will give us the ability 
+to allow developers to continue to support legacy browsers by adding the tech themselves, while allowing 
+us to minimize legacy code in video.js and decrease the footprint of the player.
+
+This follows in the footsteps of Chrome, Safari and Firefox which are all taking steps to deprecate flash.
+
+Chrome: [Chrome 56 disables flash by default]
```

Comment:
needs the extra brackets here and in the rest of the links to be linked properly